README.md

```diff
@@ -14,10 +14,33 @@ technical and performance challenges associated with the original
 code base that became apparent over the previous 10 years. Another benefit
 of the rewrite is that Volatility 3 could be released under a custom
 license that was more aligned with the goals of the Volatility community,
-the Volatility Software License (VSL). See the 
-[LICENSE](https://www.volatilityfoundation.org/license/vsl-v1.0) file for 
+the Volatility Software License (VSL). See the
+[LICENSE](https://www.volatilityfoundation.org/license/vsl-v1.0) file for
 more details.
 
+## Quick Start
+
+1. Install the required dependencies:
+
+    ```shell
+    pip install --user -e.
```

Comment:
Will this install all the dependencies, e.g. including bits needed for leehcore and s3 layers? I'm less sure on how the toml files work... I think it's just pefile but don't have a laptop to test with. 

Is it worth highlighting this is the minimum requirements and not all plugins will work? I know the various yara plugins catch people out.